.github/workflows/pypi-release.yml

```diff
@@ -0,0 +1,103 @@
+name: Build and Upload xbatcher to PyPI
+on:
+  release:
+    types:
+      - published
+  # Runs for pull requests should be disabled other than for testing purposes
+  #pull_request:
+  #  branches:
+  #    - main
```

Comment:
Do you want to test a push to test.pypi.org to make sure everything looks ok?

Reply:
Yes, will test this out tomorrow AM

Reply:
So far looks alright from https://github.com/xarray-contrib/xbatcher/actions/runs/3332266792 and https://test.pypi.org/project/xbatcher/0.1.0.post113/, but my preference would be to run a final check after merging https://github.com/xarray-contrib/xbatcher/pull/117.

Reply:
The latest run at https://github.com/xarray-contrib/xbatcher/actions/runs/3332861725 looks ok to me